xml/slert_systemd_shielding.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+
+<chapter version="5.0" xml:id="cha-shielding-with-systemd"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>Shielding with systemd</title>
+ <info/>
+ <para>
+  systemd has native support for the cpuset controller since version 244.
```

Comment:
I think, readers are probably more interested when it was first introduced. For example, if it was introduced in SLE-RT 15 SP3 or 15 SP4. The exact version here is of lesser interest IMHO. If someone really wants to know, they can inspect the changelog of the RPM package itself.

So in my humble opinion, I would change it to something like this (replace the "X" with the exact SP version):

```suggestion
  systemd has native support for the cpuset controller since &productname; 15 SPx.
```

Reply:
Version mapping would be SP3 but since it also needs cgroup v2, I specify SP4 for this.